### ISSUE TITLE
1.3-Timeline Layout-Tyler A. Reckart

### ISSUE THREAD

**Issue by tylerreckart** [TIY-GVL-FEE-2015-August/assignments#50]
### Tasks

* **Get Started**
    * [x] Open a new Issue in the class repo and add these checkboxes to your Issue.
    * [x] Create a new repository named `1.3-timeline-layout`.
    * [x] Create and commit an `index.html` file
    * [x] Push your work into Github

* [x]  **Read Learn Layout** if you haven't already

* **Write Some Code**
    * [x] Checkpoint: Initial commit!
    * [x] Checkpoint: Basic page structure (HTML only)!
    * [x] Checkpoint: First line of CSS committed and pushed!
    * [x] Checkpoint: Two column layout achieved.
    * [x] Checkpoint: Colors are exactly right.
    * [ ] Checkpoint: Icons in place.
    * [x] Checkpoint: Circles and lines finished.
    * [x] Checkpoint: Hover states and transitions are done. Fancy.
    * [x] Mission Complete

**Comment by tylerreckart** [comment]
Comfort zone: 3
Time Worked: Overall, 10 Hours. The first 4 were spent building it in a convoluted manner that I ended up trashing completely. The last 6 went into the existing product. 

Results: Almost finished. Everything is done except for the icons. Working with some SVG, however I've been working on this for 10 hours, so I'm going to bed. I'll tackle the icons in the morning. Got the whole thing to be fluid with transitions. Hover states are working good. I have noticed a slight difference in the transition fluidity between running the files locally and on Github, however that may just be my browser. 

http://tylerreckart.github.io/1.3-Timeline-Layout
